src/field/launch_application_data2.c

```diff
@@ -0,0 +1,202 @@
+#include "global.h"
+#include "alph_puzzle.h"
+#include "certificates_app.h"
+#include "choose_starter_app.h"
+#include "field/launch_application.h"
+#include "field/launch_application_data.h"
+#include "overlay_manager.h"
+#include "overlay_trainer_card.h"
+#include "overlay_trainer_card_signature.h"
+#include "overlay_14.h"
+#include "overlay_15.h"
+#include "overlay_16.h"
+#include "overlay_18.h"
+#include "overlay_41.h"
+#include "overlay_43.h"
+#include "overlay_37.h"
+#include "overlay_54.h"
+#include "overlay_55.h"
+#include "overlay_58.h"
+#include "overlay_63.h"
+#include "overlay_64.h"
+#include "overlay_65.h"
+#include "overlay_67.h"
+#include "overlay_68.h"
+#include "overlay_69.h"
+#include "overlay_70.h"
+#include "overlay_71.h"
+#include "overlay_72.h"
+#include "overlay_73.h"
+#include "overlay_86.h"
+#include "overlay_87.h"
+#include "overlay_96.h"
+#include "overlay_99.h"
+#include "overlay_100.h"
+#include "overlay_101.h"
+#include "overlay_102.h"
+#include "overlay_103.h"
+#include "overlay_104.h"
+#include "overlay_105.h"
+#include "overlay_108.h"
+#include "overlay_109.h"
+#include "overlay_111.h"
+#include "overlay_113.h"
+#include "overlay_121.h"
+#include "unk_020192D0.h"
+#include "unk_02097B78.h"
+#include "voltorb_flip/voltorb_flip.h"
+
+FS_EXTERN_OVERLAY(certificates_app);
+FS_EXTERN_OVERLAY(credits);
+FS_EXTERN_OVERLAY(trainer_card);
+FS_EXTERN_OVERLAY(trainer_card_signature);
+FS_EXTERN_OVERLAY(voltorb_flip);
+FS_EXTERN_OVERLAY(OVY_12);
+FS_EXTERN_OVERLAY(OVY_14);
+FS_EXTERN_OVERLAY(OVY_15);
+FS_EXTERN_OVERLAY(OVY_16);
+FS_EXTERN_OVERLAY(OVY_18);
+FS_EXTERN_OVERLAY(OVY_37);
+FS_EXTERN_OVERLAY(OVY_39);
+FS_EXTERN_OVERLAY(OVY_41);
+FS_EXTERN_OVERLAY(OVY_43);
+FS_EXTERN_OVERLAY(OVY_54);
+FS_EXTERN_OVERLAY(OVY_55);
+FS_EXTERN_OVERLAY(OVY_58);
+FS_EXTERN_OVERLAY(OVY_61);
+FS_EXTERN_OVERLAY(OVY_63);
+FS_EXTERN_OVERLAY(OVY_64);
+FS_EXTERN_OVERLAY(OVY_65);
+FS_EXTERN_OVERLAY(OVY_67);
+FS_EXTERN_OVERLAY(OVY_68);
+FS_EXTERN_OVERLAY(OVY_69);
+FS_EXTERN_OVERLAY(OVY_70);
+FS_EXTERN_OVERLAY(OVY_71);
+FS_EXTERN_OVERLAY(OVY_72);
+FS_EXTERN_OVERLAY(OVY_73);
+FS_EXTERN_OVERLAY(OVY_86);
+FS_EXTERN_OVERLAY(OVY_87);
+FS_EXTERN_OVERLAY(OVY_96);
+FS_EXTERN_OVERLAY(OVY_99);
+FS_EXTERN_OVERLAY(OVY_100);
+FS_EXTERN_OVERLAY(OVY_101);
+FS_EXTERN_OVERLAY(OVY_102);
+FS_EXTERN_OVERLAY(OVY_103);
+FS_EXTERN_OVERLAY(OVY_104);
+FS_EXTERN_OVERLAY(OVY_105);
+FS_EXTERN_OVERLAY(OVY_106);
+FS_EXTERN_OVERLAY(OVY_108);
+FS_EXTERN_OVERLAY(OVY_109);
+FS_EXTERN_OVERLAY(OVY_110);
+FS_EXTERN_OVERLAY(OVY_111);
+FS_EXTERN_OVERLAY(OVY_113);
+FS_EXTERN_OVERLAY(OVY_121);
+
+// _020FA444 is not referenced so the linker deadstrips it.
+// TODO: We can force the linker to include this symbol with -force_active.
```

Comment:
Or you can create a new dummy routine that includes _020FA444's data as static const, which forces it to be active due to a compiler bug. The same trick is used in math_util.c.

Reply:
Added a note to the comment